.github/workflows/full-tests-linux.yml

```diff
@@ -22,15 +22,22 @@ jobs:
         pip install pytest
         pip install pytest-cov
     - name: Get ephy_testing_data current head hash
-      id: vars
+      id: ephys
       run: echo "::set-output name=HASH_EPHY_DATASET::$(git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1)"
-    - uses: actions/cache@v2
+    - name: Cache ephys dataset - ${{ steps.ephys.outputs.HASH_EPHYS_DATASET }}
+      uses: actions/cache@v2
       id: cache-datasets
       with:
         path: /home/runner/work/nwb-conversion-tools/nwb-conversion-tools/ephy_testing_data
-        key: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-        restore-keys: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-    - name: Run pytest with coverage
+        key: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+        restore-keys: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+    - name: Set environment variables
+      run: |
+        echo "TEST_ENV=ci" >> $GITHUB_ENV
+    - name: Show environment variables
+      run: |
+        env
```

Comment:
Same question as above...

> So it looks from https://github.com/catalystneuro/nwb-conversion-tools/runs/4456719722?check_suite_focus=true that the "CI" key being used in test_gin.py is inherent to the workflow environment; thus the TEST_ENV variable here is going unused?
>
> Do we need this line then?

Now that we have the output of https://github.com/catalystneuro/nwb-conversion-tools/runs/4456719722?check_suite_focus=true to display all the built-in environment variables, anyway. Good to have as reference.